pow-migration/src/monitor.rs

```diff
@@ -100,6 +139,23 @@ fn is_valid_ready_txn(
         && txn.value == 1
 }
 
+/// Checks if the provided transaction meets the criteria in order to be
+/// considered a valid online-transaction
+fn is_valid_online_txn(txn: &TransactionDetails, block_window: &Range<u32>) -> bool {
+    if let Some(txn_data) = &txn.data {
+        if txn_data != "online" {
+            return false;
+        }
+    } else {
+        return false;
+    }
+
+    // Check for the other readiness criteria
```

Comment:
I'd have done this check first

Reply:
Im using the same format as `is_valid_ready_txn`